homeassistant/components/ozw/light.py

```diff
@@ -148,7 +148,7 @@ def async_set_duration(self, **kwargs):
 
         ozw_version = self.values.primary.ozw_instance.get_status().openzwave_version
         ozw_build = int(ozw_version.split(".")[2])
-        ozw_major = float(ozw_version[0:1])
+        ozw_major = float(".".join(ozw_version.split(".")[0:2]))
```

Comment:
We already split above. Please use that list here.

Reply:
And rename the variable to `ozw_major_minor`.

Reply:
Addressed in latest commit